ILRepack.Tests/ILRepack.Tests.csproj

```diff
@@ -55,6 +58,15 @@
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="sharpdevelop\ICSharpCode.SharpDevelop.BuildWorker120.exe.config" />
```

Comment:
I dont think we need any of these tests and test libraries. As they will only slow down the build, and not add any value.